README.md

```diff
@@ -8,7 +8,7 @@ It embraces [JSON for Linked Data (JSON-LD)](http://json-ld.org) and [Hydra Core
 
 Build a working and fully-featured CRUD API in minutes. Leverage the awesome features of the tool to develop complex and
 high performance API-first projects.
-
+<img src="https://camo.githubusercontent.com/3a20ce067d38ecc857843b57b0d926e3c718f11f/687474703a2f2f323433347a6432396d697364336534613466316537336b692e7770656e67696e652e6e6574646e612d63646e2e636f6d2f77702d636f6e74656e742f7468656d65732f6b6576696e6c656172796e65742f6173736574732f696d672f737761676765722d6c6f676f4032782e706e67" alt="Swagger" data-canonical-src="http://2434zd29misd3e4a4f1e73ki.wpengine.netdna-cdn.com/wp-content/themes/kevinlearynet/assets/img/swagger-logo@2x.png" width="50" height="50">
```

Comment:
It is the official logo. Maybe we could add the .json in the repository and validate against the validator ?